android/examples/Example_Application/app/build.gradle

```diff
@@ -14,7 +14,7 @@ android {
 }
 
 dependencies {
-    implementation 'com.ironcorelabs:ironoxide-android:0.14.1@aar'
+    implementation 'com.ironcorelabs:ironoxide-android:0.13.1@aar'
```

Comment:
We still need to retroactively fix `0.13.1`, right?

Reply:
0.13.1 is working.
0.13.2 and 0.14.1 are not.